include/crm/common/options_internal.h

```diff
@@ -62,6 +62,15 @@ _Noreturn void pcmk__cli_help(char cmd, crm_exit_t exit_code);
 void pcmk__cli_option_cleanup(void);
 
 
+/*
+ * Environment variable option handling
+ */
+
+const char *pcmk__env_option(const char *option);
+void pcmk__set_env_option(const char *option, const char *value);
+bool pcmk__env_option_enabled(const char *daemon, const char *option);
+
```

Comment:
Depending on how thorough we are with converting to glib for command line handling, there could potentially be nothing left in options_internal.h.  Another potential home for this code could be a new env.{c,h} file.

Reply:
The new files cover three sorts of options: CLI options, CIB cluster properties, and environment variable options. All of the CLI option stuff will go away with GOption but the others will both remain. I suppose they could be considered relevant to nvpair.c but that file is already quite big.